src/app/fdctl/run/tiles/fd_verify.c

```diff
@@ -199,6 +199,9 @@ unprivileged_init( fd_topo_t *      topo,
   ctx->out_wmark  = fd_dcache_compact_wmark ( ctx->out_mem, topo->links[ tile->out_link_id_primary ].dcache, topo->links[ tile->out_link_id_primary ].mtu );
   ctx->out_chunk  = ctx->out_chunk0;
 
+  /* All verify tiles use the same hashmap_seed as dedup */
+  ctx->hashmap_seed = tile->dedup.hashmap_seed;
```

Comment:
I don't think this is valid (dedup state is not present for verify tile), but see below: just initialize it in the tile itself

Reply:
fixed